src/BSON/Binary.c

```diff
@@ -90,7 +90,7 @@ static bool php_phongo_binary_init_from_hash(php_phongo_binary_t *intern, HashTa
 
 /* {{{ proto void Binary::__construct(string $data, int $type)
    Construct a new BSON binary type */
-PHP_METHOD(Binary, __construct)
+static PHP_METHOD(Binary, __construct)
```

Comment:
I spoke with @saramg and she uses this method in all of her extensions. I also found evidence of it in core (see: [here](https://php-lxr.adamharvey.name/source/search?q=%22static+PHP_METHOD%22&defs=&refs=&path=&hist=&type=&project=PHP-5.4&project=PHP-7.0)).